composer.json

```diff
@@ -15,16 +15,16 @@
         "php": "^7.2|^8.0",
         "ext-json": "*",
         "guzzlehttp/guzzle": "~7.0",
```

Comment:
not sure about the guzzlehttp/guzzle dependency.. in case it is considered a opt-in client and the lib works also with other clients it should also be moved into `require-dev` and declared as a suggested-dependency